### PR TITLE
[Fixes #97] Restyle asides on mobile

### LINKED ISSUE #97: Asides on mobile
Currently extend screen width.

Proposed solutions:  treat them as generic blocks, expand inline, slide content.

### PR DESCRIPTION
**Pull request by bluepichu**
Title.  I went with the "block following the paragraph" approach, because initial tests made every other "flashier" option seem too disruptive.

### CODE REVIEW

**Comment by el1t** [comment]
Tested working on safari responsive design and android chrome :+1:
aside

File: client/scss/media.scss [hunk 1]
```diff
@@ -40,4 +40,13 @@
 	content-row main article .author-image {
 		display: none;
 	}
+
+	content-row main article aside {
```
Comment (el1t) [inline client/scss/media.scss]: Any reason not to nest this with `content-row main article` and `content-row` above?

Comment (bluepichu) [inline client/scss/media.scss]: Nope.  I'll do that.

**Comment by el1t** [comment]
Could also include the lines starting from line 3?

**Comment by el1t** [comment]
LGTM 🎉

**Pull request merged**